apierr/error_overrides.go

```diff
@@ -33,6 +33,4 @@ var allTransientErrors = []*regexp.Regexp{
 	regexp.MustCompile(`Unknown worker environment`),
 	regexp.MustCompile(`ClusterNotReadyException`),
 	regexp.MustCompile(`worker env .* not found`),
-	regexp.MustCompile(`Timed out after `),
```

Comment:
This should not be changed. If they do, it means that you are not using the latest deco tool. Update it and rerun the generation.